src/styles/blocks/header.scss

```diff
@@ -0,0 +1,75 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  height: 60px;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px 0 $box-shadow-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  &__logo {
+    display: block;
+  }
+}
+
+.nav {
+  display: inline-block;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
```

Comment:
in order to improve your code and not to write styles like ` margin: 0; padding: 0; list-style: none;` for your list (ul) or  `text-decoration: none;` for `a` tag (link) you can create a reset.scss file and don't write the same reset styles each time

for example
```
body,
h1,
h2,
h3,
h4,
p {
  margin: 0;
}

ul {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

a {
  text-decoration: none;
}
```

after you create this file don't forget do delete unnecessary styles from your scss blocks